sdk/vision/azure-ai-vision-imageanalysis/src/samples/java/com/azure/ai/vision/imageanalysis/run_all_samples.cmd

```diff
@@ -0,0 +1,39 @@
+REM
```

Comment:
nit: we should opt for PowerShell scripts given their ability to run on all major OSes we support

Reply:
@alzimmermsft I added equivalent PowerShell scripts. Thanks!